app/src/main/java/org/schabi/newpipe/util/external_communication/ShareUtils.java

```diff
@@ -335,15 +335,7 @@ public static void copyToClipboard(@NonNull final Context context, final String
     }
 
     /**
-     * Generate a {@link ClipData} with the image of the content shared, if it's in the app cache.
-     *
-     * <p>
-     * In order not to worry about network issues (timeouts, DNS issues, low connection speed, ...)
-     * when sharing a content, only images in the {@link com.squareup.picasso.LruCache LruCache}
-     * used by the Picasso library inside {@link PicassoHelper} are used as preview images. If the
-     * thumbnail image is not in the cache, no {@link ClipData} will be generated and {@code null}
-     * will be returned.
```

Comment:
Well, this comment was there for a reason. Now it will load the thumbnail in a blocking way, so if there are network issues the share button will not work. Is there a way to access the cache manually and use a bitmap from there, like it was done before?

Reply:
Yeah, I made that change.